firebase-installations/src/main/java/com/google/firebase/installations/remote/TokenResult.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations.remote;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.auto.value.AutoValue;
+
+/** This class represents a set of values describing a FIS Auth Token Result. */
+@AutoValue
+public abstract class TokenResult {
```

Comment:
As you added this TokenResult, should we hide the proto class InstallationTokenResult only in the remote package?

Because I noticed that FirebaseInstallations class uses both.

Reply:
Both classes are needed in FirebaseInstallations. 

InstallationTokenResult as the response of the FIS SDK generateAUthToken API.
TokenResult as the response of FIS Service Client.